client/src/components/dashboardModal.js

```diff
@@ -0,0 +1,22 @@
+import React  from 'react';
+
+const DashboardModal = () => (
+  <div className="modal fade" id="myModal" tabIndex="-1" aria-labelledby="myModal" aria-hidden="true">
+    <div className="modal-dialog">
+      <div className="modal-content">
+        <div className="modal-header">
+          <h5 className="modal-title" id="exampleModalLabel">Navigating</h5>
+        </div>
+        <div className="modal-body">
+          Are you sure you want to navigate away from your current page? 
```

Comment:
can do some re-wording here

```suggestion
          <h5 className="modal-title" id="exampleModalLabel">Are you sure?</h5>
        </div>
        <div className="modal-body">
          If you leave all unsaved changes will be discarded. 
```